mne/time_frequency/tfr.py

```diff
@@ -1386,11 +1384,12 @@ def plot_joint(self, picks=None, baseline=None, mode='mean',
                             top=1. if title is not None else 1.2)
 
         # draw the connection lines between time series and topoplots
-        tf_times = [1e3 * time[0] for time in timefreqs]
-        tf_freqs = [freq[1] for freq in timefreqs]
-        lines = [_connection_line(timepoint, fig, tf_ax, map_ax_, y=freq)
-                 for timepoint, freq, map_ax_ in zip(tf_times, tf_freqs,
-                                                     map_ax)]
+        tf_times = [1e3 * time_point[0] for time_point in timefreqs]
+        tf_freqs = [freq_point[1] for freq_point in timefreqs]
```

Comment:
More pythonic:

tf_times = [1e3 * time for time, freq in timefreqs]
tf_freqs = [freq for time, freq in timefreqs]